docs/source/blog/webhook-blog.md

```diff
@@ -0,0 +1,542 @@
+            key.delete()
+    print("Finished cleaning up bucket!")
+{% endcodeblock %}
+{% enddetails %}
+<br/>
+4. Copy the cleanup script to the correct bucket prefix. From the command line, run the following:
+```bash
+aws s3 cp cleanup.py s3://showcase-bucket/script/
+```
+
+After you prepare your datasets, deploy the model pipeline in Amazon SageMaker.
+
+## Set up your model pipeline in Amazon SageMaker
+
+Create an image segmentation model pipeline in Amazon SageMaker using the ResNet50 pretrained model. Start by setting up the IAM policies needed to create and run the pipeline, then define and deploy the pipeline itself. 
+
+### Configure IAM permissions for SageMaker
+
+Set up the IAM policies for Amazon SageMaker. 
+
+1. Set up a user policy with access to Amazon SageMaker. From the command line, run the following:
+```bash
+ASSUME_POLICY=$(echo -n '{"Version":"2012-10-17","Statement":[{"Effect":"Allow","Principal":{"Service":"sagemaker.amazonaws.com"},"Action":"sts:AssumeRole"}]}')
+```
+2. Create a role to manage the SageMaker pipeline. From the command line, run the following:
+```bash
+ROLE_ARN=$(aws iam create-role \
+    --role-name SageMaker-Role \
+    --assume-role-policy-document $ASSUME_POLICY \
+    --output text \
+    --query 'Role.Arn')
+```
+3. Apply a role policy to the role that you created that grants the role full access to Amazon SageMaker. From the command line, run the following:
+```bash
+aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess --role-name SageMaker-Role
+```
+4. Attach a role policy that gives the role access to Amazon S3 so that it can retrieve the tasks and annotations from the S3 buckets and store the model output accordingly. From the command line, run the following:
+```bash
+aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess --role-name SageMaker-Role
+```
+5. Retrieve the RoleArn of the role that you just created. From the command line, run the following:
+```bash
+echo $ROLE_ARN
+```
+You need this RoleArn for the SageMaker pipeline definition. Copy the output and save it somewhere secure. 
+
+Now you're ready to define the SageMaker pipeline. 
+
+### Create a SageMaker pipeline
+
+In Amazon SageMaker, create a model training pipeline with a pretrained ResNet50 semantic image segmentation model. using the source images and annotations created in Label Studio and stored in Amazon S3. The pipeline runs the preprocessing script to split the raw and annotation data from the S3 bucket into training and validation sets, retrains the ResNet50 model, and outputs the results to the S3 bucket prefixed with `model`. 
+
+This example Amazon SageMaker pipeline definition has been updated to match the S3 bucket names that you created earlier, but you need to update it with the `RoleArn` of the role that you set up in previous steps. Replace `$SageMakerRoleArn` with the actual `RoleArn` value that you saved.
```

Comment:
Does `\"RoleArn\": \"ROLE_FOR_SAGEMAKER_PIPELINE\"` also need to be replaced with the actual RoleArn value? Yes, correct? 
@KonstantinKorotaev